loki/batch/tests/test_transformation.py

```diff
@@ -428,8 +428,8 @@ def test_transformation_file_write(tmp_path):
     ricks_path = tmp_path/'rick.roll.java'
     if ricks_path.exists():
         ricks_path.unlink()
-    FileWriteTransformation(mode='roll', suffix='.java').apply(source=source, item=item,
-                                                               build_args={'output_dir': tmp_path})
+    FileWriteTransformation(suffix='.java').apply(source=source, item=item,
```

Comment:
[no action!] The suffix should probably become an item property as well in the future, particularly when thinking of transpilation use cases?